salt/states/virt.py

```diff
@@ -265,7 +265,10 @@ def running(name,
             password=None,
             os_type=None,
             arch=None,
-            boot=None):
+            boot=None,
+            serial_type=None,
+            console=None,
+            telnet_port=None):
```

Comment:
As mentioned in the `virt.update` change, those parameters should be deprecated in favor of `serials` and `consoles`